### PR TITLE
give Cylc install the correct options

### PR DESCRIPTION
**Pull request by wxtim**
Fix test failures.

### CODE REVIEW

File: cylc/rose/stem.py [hunk 1]
```diff
@@ -488,7 +486,7 @@ def main():
     opts = StemRunner(opts).process()
 
     # call cylc install
-    cylc_install(parser, opts, args)
+    cylc_install(parser, opts)
```
Comment (oliver-sanders) [inline cylc/rose/stem.py:489]: Ah yes, that would have been https://github.com/cylc/cylc-flow/pull/4132

**Comment by oliver-sanders** [comment]
The three pre_configure tests still seem to be failing:

FAILED tests/functional/test_pre_configure.py::test_validate[06_jinja2_thorough-envvars5-None]
482
FAILED tests/functional/test_pre_configure.py::test_validate[09_template_vars_vanilla-envvars7-None]
483
FAILED tests/functional/test_pre_configure.py::test_process[06_jinja2_thorough-envvars4-None]

**Comment by wxtim** [comment]
I'm not totally happy about the fix, but I think that it is sufficient.

**Review by oliver-sanders — APPROVED** [review]
Not too sure what's going on with the whitespace but happy tests are passing.

**Review by oliver-sanders — CHANGES_REQUESTED** [review]
Sorry, just spotted something.

The `REG` argument in `rose stem [REG]` is now totally ignored (it was ignored by `cylc install` anyhoo). Should remove it, the `--flow-name` argument fills this function now?

**Review by datamel — COMMENTED** [review]
I have read code and run tests, all passing for me locally. 

This is coming from your master branch btw :)

**Comment by wxtim** [comment]
> This is coming from your master branch btw :)

Ooops. Ah Well

**Comment by oliver-sanders** [comment]
> This is coming from your master branch btw :)

That's alright, master is just another branch.

**Comment by wxtim** [comment]
> Sorry, just spotted something.
> 
> The `REG` argument in `rose stem [REG]` is now totally ignored (it was ignored by `cylc install` anyhoo). Should remove it, the `--flow-name` argument fills this function now?

Where did you spot it - I can only see it in the help docs...

File: cylc/rose/stem.py [hunk 2]
```diff
@@ -17,7 +17,12 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""Implementation of 'rose stem'"""
+"""Cylc 8 implementation of 'rose stem'
+
+Provide a wrapper around ``cylc install`` for rose-stem suites.
+
+To run a rose-stem suite use ``cylc play``.
```
Comment (oliver-sanders) [inline cylc/rose/stem.py:24]: This isn't RST:

```suggestion
Provide a wrapper around "cylc install" for rose-stem suites.

To run a rose-stem suite use "cylc play".
```

**Pull request merged**